src/__tests__/server.test.ts

```diff
@@ -74,3 +76,40 @@ test('stop twice does nothing on second attempt', async () => {
 
   fail('expected a connection error as server should be closed');
 });
+
+const getPort = async (options?: Options) => {
+  const { schema, ctxFactory } = testGraphql(options);
+  ctxFactory.config.port = await availablePort();
+
+  await startServer(schema, ctxFactory);
+
+  const address = getAddress();
+  await healthcheckOk;
+  return address.port;
+};
+
+const ropstenOptions: Options = {
```

Comment:
No need to have these options outside the scope of the test, they can be inside the function that uses them.